tests/php/GeometryTest.php

```diff
@@ -21,6 +21,17 @@ class GeometryTest extends GeographyTest
         'Within' => ['Equals', 'Within'],
     ];
 
+    protected static $mariadb_test_methods = [
```

Comment:
I am not 100% sure what the test that references this variable and `$mysql_test_methods` is doing, checking database capabilities?  This is what mariadb spits out